src/main/java/com/google/api/gax/grpc/PageStreamingCallable.java

```diff
@@ -60,9 +60,11 @@ public String toString() {
 
   @Override
   public ListenableFuture<PagedListResponse<RequestT, ResponseT, ResourceT>> futureCall(
-      CallContext<RequestT> context) {
+      CallContext<RequestT, PagedListResponse<RequestT, ResponseT, ResourceT>> context) {
+    CallContext<RequestT, ResponseT> newContext =
+        CallContext.of(context.getChannel(), context.getCallOptions(), context.getRequest());
     PagedListResponse<RequestT, ResponseT, ResourceT> pagedListResponse =
-        new PagedListResponseImpl<>(callable, pageDescriptor, context);
+        new PagedListResponseImpl<>(callable, pageDescriptor, newContext);
```

Comment:
Why are you creating a new context? It's not a proper copy since it is skipping StreamingObserver.